asapdiscovery-genetics/asapdiscovery/genetics/scripts/make_alignment.py

```diff
@@ -0,0 +1,46 @@
+from asapdiscovery.genetics.blast import PDBentry, get_blast_seqs
+from asapdiscovery.genetics.seq_alignment import Alignment, do_MSA
+
+
```

Comment:
You can turn this into a command line script by adding CLI args like so: 
See https://github.com/choderalab/asapdiscovery/blob/main/asapdiscovery-dataviz/asapdiscovery/dataviz/scripts/pose_to_viz.py for a good example. 
```python
parser = argparse.ArgumentParser(description="Turn a trajectory into a GIF")

parser.add_argument(
    "--fasta",
    type=str,
    required=True,
    help="Path to the fasta file",
)

  args = parser.parse_args()
  # use args.fasta ... etc
```

and adding an entrypoint in `asapdiscovery-genetic/pyproject.toml`

like this:

```toml
[project.scripts]
make-alignment = "asapdiscovery.genetic.scripts.make_alignment:main"
```

You will also need to change the make_alignment function to `main` and add 
```python
if __name__ == "__main__":
    main()
```
At the bottom of the file.

Reply:
The you should be able to call it as `make-alignment --fasta myfasta.fasta` etc etc.